pkg/controllers/management/gke/gke_cluster_handler.go

```diff
@@ -246,6 +246,18 @@ func (e *gkeOperatorController) onClusterChange(key string, cluster *mgmtv3.Clus
 		e.ClusterEnqueueAfter(cluster.Name, enqueueTime)
 		if failureMessage == "" {
 			logrus.Infof("waiting for cluster GKE [%s] to update", cluster.Name)
+
+			// If the HealthSyncer runs while upgrading a zonal cluster, the control plane may not be reachable.
+			// This adds additional context to the error message to help explain that this is normal.
+			readyMsg := apimgmtv3.ClusterConditionReady.GetMessage(cluster)
```

Comment:
Would this message not be found on the updating condition?

Reply:
No, it's only the Ready condition that gets the health check message set

https://github.com/rancher/rancher/blob/f5ff8e1fbe76e63d2197185c0d7adda72bb1db70/pkg/controllers/managementuser/healthsyncer/healthsyncer.go#L118
https://github.com/rancher/rancher/blob/f5ff8e1fbe76e63d2197185c0d7adda72bb1db70/pkg/controllers/managementuser/healthsyncer/healthsyncer.go#L80